__init__.py

```diff
@@ -161,7 +161,6 @@ def create_compify_material(name, camera, footage):
     mat = bpy.data.materials.new(name)
     mat.use_nodes = True
     mat.blend_method = 'HASHED'
-    mat.shadow_method = 'HASHED'
```

Comment:
Instead of removing this, you could create a function that will work in both environments:

```python
def _set_hashed_shadow(mat):
    """
    Make a material cast alpha‑hashed (dithered) shadows on
    every Blender version that still supports the add‑on.
    """
    if hasattr(mat, "shadow_method"):                       # Blender 2.8 – 3.6
        mat.shadow_method = 'HASHED'
    elif hasattr(mat, "surface_render_method"):             # Blender 4.0 +
        # ‘DITHERED’ == hashed transparency in Eevee‑Next
        mat.surface_render_method = 'DITHERED'
```
Than call that in place of the removed line. This seems to work as far as 5.0.